barretenberg/cpp/src/barretenberg/api/log.hpp

```diff
@@ -43,15 +43,15 @@ inline void writeUint64AsRawBytesToStdout(uint64_t value)
         value >>= 8;
     }
 
-    writeRawBytesToStdout(bytes);
+    write_bytes_to_stdout(bytes);
 }
 
 /**
  * @brief Writes a sting to stdout
  *
  * @param str The raw string to write to stdout
  */
-inline void writeStringToStdout(const std::string& str)
+inline void write_string_to_stdout(const std::string& str)
```

Comment:
can we just delete this and replace every use with `std::cout << str;`

Reply:
(answer can be no)

Reply:
Done